packages/block-editor/src/store/reducer.js

```diff
@@ -1768,25 +1778,9 @@ export function automaticChangeStatus( state, action ) {
 			}
```

Comment:
Should we move this code to the higher-order reducer and get rid of this reducer all together (to avoid logic in two places)?

Reply:
I guess if it's allowed to add a property without having a reducer